scim/users.go

```diff
@@ -88,6 +89,18 @@ func (a UsersAPI) Delete(userID string) error {
 	return a.client.Scim(a.context, http.MethodDelete, userPath, nil, nil)
 }
 
+func (a UsersAPI) DeleteRepos(userID string) error {
+	repoPath := fmt.Sprintf("/Repos/%v", userID)
+	recursive := true
```

Comment:
why do you extract variables if they're used only once and don't aid readability?

Reply:
I thought they do aid readability. Will remove